docs/source/triggers/index.rst

```diff
@@ -14,81 +14,63 @@ and the UI, Daml applications often need to automate certain
 template and choice types in Daml and a representation of those Daml
 types in the language you are using for your client. Daml triggers
 address this problem by allowing you to write certain kinds of
-automation directly in Daml reusing all the Daml types and logic that
-you have already defined. Note that while the logic for Daml triggers
-is written in Daml, they act like any other ledger client: They are
+automation directly in Daml, reusing all the Daml types and logic that
+you have already defined. Note that, while the logic for Daml triggers
+is written in Daml, they act like any other ledger client: they are
 executed separately from the ledger, they do not need to be uploaded
 to the ledger and they do not allow you to do anything that any other
 ledger client could not do.
 
-Usage
-=====
+Sample Trigger
+==============
 
-Our example for this tutorial consists of 3 templates.
+Our example for this tutorial builds upon the Getting Started Guide,
+specifically picking up right after the :doc:`/getting-started/first-feature`
+section.
 
-First, we have a template called ``Original``:
+We're going to build a chatbot that answers every message with
 
-.. literalinclude:: ./template-root/src/CopyTrigger.daml
-   :language: daml
-   :start-after: -- ORIGINAL_TEMPLATE_BEGIN
-   :end-before: -- ORIGINAL_TEMPLATE_END
+  "Please, tell me more about that."
 
-This template has an ``owner``, a ``name`` that identifies it and some
-``textdata`` that we just represent as ``Text`` to keep things simple.  We
-have also added a contract key to ensure that each owner can only have
-one ``Original`` with a given ``name``.
+That should fool anyone and pass the Turing test, easily.
 
-Second, we have a template called ``Subscriber``:
 
-.. literalinclude:: ./template-root/src/CopyTrigger.daml
-   :language: daml
-   :start-after: -- SUBSCRIBER_TEMPLATE_BEGIN
-   :end-before: -- SUBSCRIBER_TEMPLATE_END
-
-This template allows the ``subscriber`` to subscribe to ``Original`` s where ``subscribedTo`` is the ``owner``.
-For each of these ``Original`` s, our Daml trigger should then automatically create an instance of
-third template called ``Copy``:
-
-.. literalinclude:: ./template-root/src/CopyTrigger.daml
-   :language: daml
-   :start-after: -- COPY_TEMPLATE_BEGIN
-   :end-before: -- COPY_TEMPLATE_END
-
-Our trigger should also ensure that the ``Copy`` contracts stay in sync with changes on the ledger. That means
-that we need to archive ``Copy`` contracts if there is more than one for the same ``Original``, we need to archive
-``Copy`` contracts if the corresponding ``Original`` has been archived and we need to archive
-all ``Copy`` s for a given subscriber if the corresponding ``Subscriber`` contract has been archived.
-
-Implementing a Daml Trigger
----------------------------
+Daml Trigger Basics
+===================
 
 Having defined what our Daml trigger is supposed to do, we can now
```

Comment:
this sentence sounds very weird now given that 1. the explanation has shrunk down to a single sentence now and 2. it’s no longer "the trigger should" but "we will build a trigger". Not quite sure how to rephrase it.

Reply:
I think just removing the sentence works.